src/Fixer/FunctionNotation/UseArrowFunctionsFixer.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\FunctionNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\VersionSpecification;
+use PhpCsFixer\FixerDefinition\VersionSpecificCodeSample;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Tokenizer\TokensAnalyzer;
+
+/**
+ * @author Gregor Harlan
+ */
+final class UseArrowFunctionsFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Anonymous functions with one-liner return statement must use arrow functions.',
+            [
+                new VersionSpecificCodeSample(
+                    <<<'SAMPLE'
+<?php
+foo(function ($a) use ($b) {
+    return $a + $b;
+});
+
+SAMPLE
+                    ,
+                    new VersionSpecification(70400)
+                ),
+            ],
+            null,
+            'Risky when using "isset()" on outside variables that are not imported with "use ()".'
```

Comment:
can you give an example of this? I'm curious because I haven't looked into all the edge cases of the arrow function yet

Reply:
```php
$foo = 1;

bar(function () {
    return isset($foo) ? 2 : 3;
});
// the func returns 3

bar(fn () => isset($foo) ? 2 : 3);
// the func returns 2
```

Reply:
thanks for the sample, all clear now :)

Reply:
Is that the only risky scenario?

Reply:
There are some variants of the same issue, like `empty()` and `??` and:

```php
$foo = [1];

bar(function () {
    return ($foo[] = 2) ? $foo : null;
});

bar(fn () => ($foo[] = 2) ? $foo : null);
```
https://3v4l.org/RkfNu#v740